fs/copy.py

```diff
@@ -56,13 +88,35 @@ def copy_file(src_fs, src_path, dst_fs, dst_path):
             if src_fs is dst_fs:
                 # Same filesystem, so we can do a potentially optimized copy
                 src_fs.copy(src_path, dst_path, overwrite=True)
+                return dst_path
             else:
                 # Standard copy
                 with src_fs.lock(), dst_fs.lock():
                     with src_fs.open(src_path, 'rb') as read_file:
                         # There may be an optimized copy available on dst_fs
                         dst_fs.setbinfile(dst_path, read_file)
+                        return dst_path
 
+def copy_file_if_newer(src_fs, src_path, dst_fs, dst_path):
```

Comment:
See comment above! ;-)
No worries, I will add the docs to all functions once we agree on stable code base!